src/spatialref.jl

```diff
@@ -616,6 +617,31 @@ function toPROJ4(spref::AbstractSpatialRef)::String
     return unsafe_string(projptr[])
 end
 
+"""
+    toEPSG(spref::AbstractSpatialRef)
+
+If possible, export EPSG code for this coordinate system.
+"""
+function toEPSG(spref::AbstractSpatialRef)::String
```

Comment:
EPSG is a code, so this should return an Int? GFT.EPSG(1) does also work, so it should pose no further changes.

Reply:
I'm not sure what we want to do here... changing this to an Int won't be a prblem for GFT but are there other applicaitons that use crs text?

Reply:
Well you introduced this function here, so no one uses it yet.

Reply:
Now updated to return an Int32